### ISSUE TITLE
Annoying error message fix - diff2html.sh :  line 121

### ISSUE THREAD

**Issue by patricksereno** [ckujau/scripts#2]
diff2html.patch
````
--- /old/diff2html.sh	2020-02-06 15:56:17.000000000 -0700
+++ /new/diff2html.sh	2020-02-06 15:24:27.000000000 -0700
@@ -118,7 +118,7 @@
 done
 IFS=$OIFS

-if [ $diffseen -eq 0  &&  $onlyseen -eq 0 ]; then
+if [[ $diffseen -eq 0  &&  $onlyseen -eq 0 ]]; then
 	printf '</span>'
 else
 	echo "</div>"
````

**Comment by ckujau** [comment]
So, the whole error message is actually:

    ./diff2html.sh: line 121: [: missing `]'

While the _double brackets_ are supported in `bash` and [may even be the better choice here](https://unix.stackexchange.com/a/32211/31256 "Why does parameter expansion with spaces without quotes work inside double brackets"), but I don't like non-POSIX expressions, so the following should help too:

```
--- a/diff2html.sh
+++ b/diff2html.sh
@@ -118,7 +118,7 @@ while [ $? -eq 0 ]; do
 done
 IFS=$OIFS
 
-if [ $diffseen -eq 0  &&  $onlyseen -eq 0 ]; then 
+if [ $diffseen -eq 0 ] && [ $onlyseen -eq 0 ]; then 
        printf '</span>'
 else
        echo "</div>"
````

But yeah, all these scripts here need some TLC, so thanks for bringing this up!

**Comment by patricksereno** [comment]
Good idea… your work is great!  Many thanks!

**Comment by ckujau** [comment]
Fixed with 2541da5482c5b0db1b7c88490e28d8897313bbc8
